Src/BootCamp.Chapter/Program.cs

```diff
@@ -11,6 +11,58 @@ class Program
     {
         static void Main(string[] args)
```

Comment:
I think you could extract the code to another function. This keeps your Main small and clean.

Reply:
solved